cmd/init-repo/main.go

```diff
@@ -164,22 +167,45 @@ func cloneForkRepo(cfg config.Config, repoName string) {
 	run(setEmailCmd)
 }
 
+// installGodeps installs kubernetes' forked version of godep.
+// We need to install the forked version because godep by default
+// doesn't support bitbucket anymore, but the forked version does.
+// Since the forked godep only exists until 1.14, we first checkout
+// to a commit which supports it.
 func installGodeps() {
-	if _, err := exec.LookPath("godep"); err == nil {
-		glog.Infof("Already installed: godep")
+	godepVersionCmd := exec.Command("godep", "version")
+	version, err := godepVersionCmd.Output()
+	if err == nil && string(version) == godepVersion {
+		glog.Infof("Already installed godep %s", godepVersion)
 		return
 	}
-	glog.Infof("Installing github.com/tools/godep#%s ...", godepCommit)
-	run(exec.Command("go", "get", "github.com/tools/godep"))
 
-	godepDir := filepath.Join(SystemGoPath, "src", "github.com", "tools", "godep")
-	godepCheckoutCmd := exec.Command("git", "checkout", godepCommit)
-	godepCheckoutCmd.Dir = godepDir
-	run(godepCheckoutCmd)
+	// clone k8s.io/kubernetes if it doesn't exist already
+	if _, err := os.Stat(filepath.Join(SystemGoPath, "src", "k8s.io", "kubernetes")); err != nil {
```

Comment:
This is needed for bot instances that don't have kubernetes installed already. I only know of openshift that uses the publishing-bot, not sure if there are others as well.

Also not sure if openshift runs a godeps version of the bot? cc @sttts @mfojtik